src/content/en/2020/markup.md

```diff
@@ -626,34 +704,42 @@ Using `target="_blank"` is known as a [security vulnerability](https://mathiasby
 <figcaption>{{ figure_link(caption="Blank relationships.", sheets_gid="1876528165", sql_file="pages_wpt_bodies_by_device.sql") }}</figcaption>
 </figure>
 
-As a rule of thumb—also [for usability reasons](https://www.nngroup.com/articles/new-browser-windows-and-tabs/)—, prefer not to use `target=_blank`. 
+As a rule of thumb and for [usability reasons](https://www.nngroup.com/articles/new-browser-windows-and-tabs/), prefer not to use `target="_blank"`. 
 
-<p class="note">Within the latest Safari and Firefox versions, setting <code>target="_blank"</code> on <code>a</code> elements implicitly provides the same <code>rel</code> behavior as setting <code>rel="noopener"</code>. This is currently being <a href="https://bugs.chromium.org/p/chromium/issues/detail?id=898942">implemented in Chromium</a> as well.</p>
+<p class="note">Within the latest Safari and Firefox versions, setting <code>target="_blank"</code> on <code>a</code> elements implicitly provides the same <code>rel</code> behavior as setting <code>rel="noopener"</code>. This is already <a href="https://chromium-review.googlesource.com/c/chromium/src/+/1630010">implemented in Chromium</a> as well and will land in Chrome 88.</p>
 
+{# TODO(editors, authors): The following two sections each read like conclusions. Consider merging. #}
```

Comment:
I also feel something still needs to be done here. Merging the section may require a bigger rewrite… Is there something elegant we could do with the heading here, and rename it from “Status and trends” to just “Trends”? “Observations”?

Reply:
I tend to agree with Rick that I'm confused with this section. It seems to be repeating some stats highlighted previously, yet we have a separate Conclusion where you'd normally expect a summary to be.

I'd merge it, but also move the last section "We're leaving this open to you. What are your observations? ... Leave a comment to share your thoughts!" to the very end. It's a nice way to hand over the chapter to the user and also leads on to the new "Explore the results" heading.

If you merge the first two paragraphs of the current Conclusion ("When we don't need to..." and "Yet what we observe with this year's data"), I think they lead on quite nicely from the "Proprietary product-specific elements..." paragraph in this section, so it reads as one summary conclusion. Slightly long, but that's OK.

Reply:
Just playing with options, what speaks against “Observations”? It's not perfect yet, but it seems to be an improvement that allows to cover very specific notes in this former “Status” section, and shift to general commentary in the conclusion.

Reply:
You still end up with two conclusions by different names :/

Reply:
…I think @bazzadp’s approach could work. I had to look at this [in document form](https://docs.google.com/document/d/1YR9HpqOPOCy7eYQC2IngzrqjPmYLSFoPCL_W8qyXJw4/edit#) to get a better feel for and play with it (paragraphs I edited for this are marked)… while I want to read through this again, what do you think?